CONTRIBUTING.md

```diff
@@ -0,0 +1,240 @@
+# Contributing to Postman Newman
+
+  - [Getting Started Quick](#getting-started-quick)
+  - [NPM Command Reference](#npm-command-reference)
+      - [`npm install`](#npm-install)
+      - [`npm test`](#npm-test)
+      - [`npm run test-system`](#npm-run-test-system)
+      - [`npm run test-lint`](#npm-run-test-lint)
+      - [`npm run test-unit`](#npm-run-test-unit)
+      - [`npm run test-integration`](#npm-run-test-integration)
+  - [Repository](#repository)
+      - [Structure](#structure)
+      - [Branching and Tagging Policy](#branching-and-tagging-policy)
+  - [Preferred IDE](#preferred-ide)
+  - [Commit Guidelines](#commit-guidelines)
+      - [Check for errors before committing](#check-for-errors-before-committing)
+      - [Atomic commits](#atomic-commits)
+      - [Clean commit message](#clean-commit-message)
+  - [Guidelines for sending a Pull Request](#guidelines-for-sending-a-pull-request)
+  - [Documentation guidelines](#documentation-guidelines)
+  - [The CI Platform](#the-ci-platform)
+      - [Ensuring your commits will not fail build](#ensuring-your-commits-will-not-fail-build)
+      - [Accessing build log on CI server](#accessing-build-log-on-ci-server)
+  - [Security](#security)
+
+## Getting Started Quick
+
+In order to contribute to this project, you should:
+
+1. Clone this repository / your fork of this repository using `git`
+2. If not on `develop`, switch to the `develop` branch with `git checkout develop`
+3. Run `npm install` in the project directory
+4. Make sure everything is working by running `npm test`.
+5. Create a new feature branch from `develop` with `feature/your-feature-name`.
+6. Make the necessary changes in line with the objective(s) of the pull request, and
```

Comment:
add a point after 6, saying

"ensure that feature additions and bug fixes are accompanied by their own unit and integration tests"